src/notation/internal/notationconfiguration.cpp

```diff
@@ -164,6 +165,10 @@ void NotationConfiguration::init()
     settings()->setDefaultValue(IS_METRONOME_ENABLED, Val(false));
     settings()->setDefaultValue(IS_COUNT_IN_ENABLED, Val(false));
 
+    settings()->setDefaultValue(PLAYBACK_SMOOTH_PANNING, Val(false));
+    settings()->setDescription(PLAYBACK_SMOOTH_PANNING, qtrc("notation", "Smooth panning").toStdString());
```

Comment:
You can replace `qtrc("notation", "Smooth panning").toStdString()` with `trc("notation", "Smooth panning")`